scripts/docdiff

```diff
@@ -11,4 +11,5 @@ if [ ! -f target/release/docdiff ]; then
   cd ../..
 fi
 
-./target/release/docdiff s2n-quic
+# dont diff unstable apis
```

Comment:
That's not really what the flag does, right? It seems to just disable the compile error